app/com/linkedin/drelephant/tuning/AutoTuningAPIHelper.java

```diff
@@ -80,6 +83,21 @@ private JobSuggestedParamSet getBestParamSet(String jobDefId) {
     return jobSuggestedParamSetBestParamSet;
   }
 
+  /**
+   * For a job, returns the Manually tuned param set
+   * @param jobDefId Sting JobDefId of the job
+   * @return JobSuggestedParamSet the best parameter set of the given job if it exists else  the default parameter set
+   */
+  private JobSuggestedParamSet getManuallyTunedParamSet(String jobDefId) {
+    JobSuggestedParamSet jobSuggestedParamSetBestParamSet = JobSuggestedParamSet.find.select("*")
```

Comment:
If the columns changes in job_suggested_param_set table , then developers  also have to change  JobSuggestedParamSet class . Hence I don't think , it may cause any issue